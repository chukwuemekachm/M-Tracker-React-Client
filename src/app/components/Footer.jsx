import React from 'react';
import '../assets/css/footer.css';

/**
 * @description The Footer component
 *
 * @returns {object}
 */
const Footer = () => (
  <div className="w3-flat-midnight-blue footer">
    Copyright of &copy;2018 Maintenance Tracker by Chima Chukwuemeka
  </div>
);

export default Footer;
