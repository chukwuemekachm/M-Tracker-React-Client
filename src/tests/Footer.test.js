import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../app/components/Footer';

describe('Footer component', () => {
  it('should render without errors', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find('div')).toHaveLength(1);
    expect(wrapper).toMatchSnapshot();
  });
});
