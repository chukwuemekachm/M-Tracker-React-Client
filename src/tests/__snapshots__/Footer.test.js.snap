// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render without errors 1`] = `
<div
  className="w3-flat-midnight-blue footer"
>
  Copyright of Â©2018 Maintenance Tracker by Chima Chukwuemeka
</div>
`;
